clib/exninfo.ml

```diff
@@ -104,3 +117,19 @@ let info e =
     (* Mismatch: the raised exception is not the one stored, either because the
        previous raise was not instrumented, or because something went wrong. *)
     Store.empty
+
+let debug_reraise = false
+
+let capture e =
+  if !is_recording then
+    (* This must be the first function call, otherwise the stack may be
+       destroyed *)
+    let bt = Printexc.get_raw_backtrace () in
+    if debug_reraise then
+      (Printexc.print_raw_backtrace stderr bt;
+       output_string stderr "--------------------------------\n";
+       flush stderr);
```

Comment:
I don't think this should stay in production code.

Reply:
Removed.